BoxContentSDK/BoxContentSDK/Requests/BOXRequest_Private.h

```diff
@@ -71,6 +72,12 @@
                                  successBlock:(BOXAPIHeaderSuccessBlock)successBlock
                                  failureBlock:(BOXAPIHeaderFailureBlock)failureBlock;
 
+- (BOXAPIOAuth2ToJSONOperation *)authOperationWithURL:(NSURL *)URL
```

Comment:
Do you mean for this header? This is basically where a lot of the inner-working pieces get defined, which aren't generally needed (so not in the public .h) but may be used for advanced configuration, etc.
We can find an even better place for this stuff, but currently the convention is that these operation setup helpers all live in here. What do you think?